### PR TITLE
Create page for editing

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/component/fieldsForBook/FieldsForBook.tsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    errorsValid: ErrorType[] | null,
+    messageCreated: string,
+    isShowSnackbar: boolean,
+    onCloseSnackbar: () => void,
+    handlerCreateBook: () => Promise<void>,
+    buttonCreateName: string,
+    isEdit?: boolean,
+}
+  
+export const FieldsForBook: FC<Props> = ({ namePage, form, isEdit, setFormBook, loading, errorsValid, messageCreated, isShowSnackbar, onCloseSnackbar, handlerCreateBook, buttonCreateName }) => {
```
Comment (vell28) [inline src/component/fieldsForBook/FieldsForBook.tsx]: Rename `FieldsForBook` to `BookForm` and file as well

File: src/utils/addFieldForBook.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { ChangeEvent } from "react"
+
+import { IBook } from "../interface/IBook"
+import { parceStringToNumberAndPoint } from "./parceStringToNumberAndPoint"
+
+export const addFieldForBook = (
```
Comment (vell28) [inline src/utils/addFieldForBook.ts]: addFieldForBook  => addFieldToBook

File: src/pages/createBook/CreateBook.tsx [hunk 3]
```diff
@@ -1,47 +1,26 @@
...
     
     const { loading, request, errorsValid } = useHttp()
     
-    const { isOpen: isShowSnackbar, onToggle: toggleShowSnackbar } = useToggle()
-    
-    const closeCreatePage = useNavigate()
-
-    const handlerChange = useCallback(({ target }: ChangeEvent<HTMLInputElement>): void => {
-        let newValue: File | string = target?.value
+    const setFormBook = useCallback((target: ChangeEvent<HTMLInputElement>) => {
```
Comment (vell28) [inline src/pages/createBook/CreateBook.tsx]: remove `{}`

File: src/component/fieldsForBook/FieldsForBook.tsx (hunk 1 above)
Comment (Ksenaaa) [reply src/component/fieldsForBook/FieldsForBook.tsx]: fixed

File: src/utils/addFieldForBook.ts (hunk 2 above)
Comment (Ksenaaa) [reply src/utils/addFieldForBook.ts]: fixed

File: src/pages/createBook/CreateBook.tsx (hunk 3 above)
Comment (Ksenaaa) [reply src/pages/createBook/CreateBook.tsx]: fixed

**Pull request merged**